magicl.asd

```diff
@@ -139,10 +139,12 @@
 (defmethod perform ((operation compile-op) (component f->so))
   (flet ((nn (x) (uiop:native-namestring x)))
     (let* ((fortran-file (component-pathname component))
-           (object-file (make-pathname :type "o" :defaults fortran-file))
+           (object-file (apply-output-translations
+			 (make-pathname :type "o" :defaults fortran-file)))
```

Comment:
This doesn't appear to be indented correctly (tabs v spaces?)

Reply:
Oops! Fixed.

Reply:
It still looks like there are tabs, but I haven't verified by downloading the source. The indentation still isn't right.

Sorry, I know this is so trivial. I'd edit your branch if I could.